lxd/instance/drivers/driver_qemu.go

```diff
@@ -350,6 +333,30 @@ func qemuCreate(s *state.State, args db.InstanceArgs, volumeConfig map[string]st
 	return d, nil
 }
 
+func (d *qemu) getRootDiskDevice() (string, map[string]string, error) {
```

Comment:
Is this function common to both instance types? If so could you move it to the common driver?